snuba/clickhouse/formatter/expression.py

```diff
@@ -202,7 +202,12 @@ def _format_date_literal(self, exp: Literal) -> str:
         )
 
 
-class ClickHouseExpressionFormatterAnonymized(ClickhouseExpressionFormatterBase):
+def _gen_random_number() -> int:
+    # https://xkcd.com/221/
+    return 4
```

Comment:
I think we should have a design meeting and set up a committee to define a process to choose this number.